my-ansible-inventory

```diff
@@ -0,0 +1,3 @@
+ini
```

Comment:
I think having the `ini` here is a typo, or at least unnecessary though not an issue